alacritty/src/input.rs

```diff
@@ -623,8 +622,7 @@ impl<T: EventListener, A: ActionContext<T>> Processor<T, A> {
                 MouseButton::Left => 0,
                 MouseButton::Middle => 1,
                 MouseButton::Right => 2,
-                // Can't properly report more than three buttons.
-                MouseButton::Other(_) => return,
+                MouseButton::Other(x) => x.try_into().unwrap(),
```

Comment:
I really doubt this reports mouse buttons correctly. Did you just guess at what value should be reported?

Reply:
Hi @chrisduerr  thanks for your time. I have tested this with my mouse that has one additional button (button 8) and the number reported is 8. Then I used xbindkeys to bind button 9, 10 and 11 with my left shift and the value reported was 9, 10 and 11 respectively.  Should I follow the convention linked by you ? (https://invisible-island.net/xterm/ctlseqs/ctlseqs.html#h3-Other-buttons)

Reply:
Yes you should.